src/components/routerView.vue

```diff
@@ -24,6 +25,8 @@
   const route = useRoute()
   const rejection = useRejection()
   const depth = useRouterDepth()
+
+  const propStore = inject(propStoreKey)
```

Comment:
Nit: be nice to have a composition for this. usePropsStore or something.

Reply:
And it can throw a router not installed error rather than being possibly undefined.

Reply:
👍